src/site/_includes/js/eligibility.js

```diff
@@ -2647,6 +2664,34 @@ function wicResult(input) {
   return program.getResult();
 }
 
+function clipperStartResult(input) {
+  const meetsAgeReq = and(
+    ge(input.age, cnst.clipper.MIN_ELIGIBLE_AGE),
+    le(input.age, cnst.clipper.MAX_ELIGIBLE_AGE));
+
+  const grossLimit = MonthlyIncomeLimits.fromAnnual(
+    cnst.clipper.ANNUAL_INCOME_LIMITS,
+    cnst.clipper.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
+
+  const incomeLimit = grossLimit.getLimit(input.householdSize);
+  const underIncomeLimit = le(grossIncome(input), incomeLimit);
+
+  const noRTC = not(input.existingRtcClipperMe);
+
+  const program = new Program();
+  program.addCondition(new EligCondition(
+    'Be 19-64 years old',
```

Comment:
To be sure this text stays in sync with any future program eligibility changes, use the constants you defined for the ages.  You can use a string template literal, e.g.

```
`Be ${cnst.clipper.MIN_ELIGIBLE_AGE} - ${cnst.clipper.MAX_ELIGIBLE_AGE} years old`
```

Reply:
Done in [214111f](https://github.com/theunitedeffort/theunitedeffort.org/pull/610/commits/214111fe470cd9cb21922e016adc138278e99ed8)